examples/aws-lambda-dynamodb-trigger/aws-lambda-dynamodb-trigger.md

```diff
@@ -0,0 +1,50 @@
+# AWS Lambda DynamoDB trigger
+
+This example creates a function, which will be executed for each entry added to a database in the [DynamoDB](https://aws.amazon.com/dynamodb/).
+
+For more information, see the [AWS Lambda learn guide](https://ballerina.io/learn/run-in-the-cloud/function-as-a-service/aws-lambda/).
+
+## Set up the prerequisites
+
+For instructions, see [Set up the prerequisites](https://ballerina.io/learn/run-in-the-cloud/function-as-a-service/aws-lambda/#set-up-the-prerequisites).
+
+## Write the function
+
+Follow the steps below to write the function.
+
+1. Execute the command below to create a new Ballerina package.
+
+::: out bal_new.out :::
+
+2. Replace the content of the generated Ballerina file with the content below.
+
+::: code aws-lambda-dynamodb-trigger.bal :::
+
+## Build the function 
+
+Execute the command below to generate the AWS Lambda artifacts.
+
+::: out bal_build.out :::
+
+## Deploy the function
+
+Execute the AWS CLI command given by the compiler to create and publish the functions by replacing the respective AWS `$LAMBDA_ROLE_ARN`, `$REGION_ID`, and `$FUNCTION_NAME` values given in the command with your values.
+
+::: out aws_deploy.out :::
+
+## Invoke the function
+
+Follow the instructions below to create a DynamoDB table for invoking this function.
+
+1. In the IAM Console, click the corresponding role in the list, and click **Add permissions**.
+2. Select **attach policies** from the drop-down menu, and add the **AWSLambdaDynamoDBExecutionRole** to the role.
+3. Go to [DynamoDB](https://us-west-1.console.aws.amazon.com/dynamodbv2), and from the drop-down menu at the RHS of the screen, select the **AWS region** in which you created the user and role.
```

Comment:
Is it fine to have `us-west-1` part in URL @xlight05 ?

Reply:
Added the DynamoDB Console URL.

Reply:
Not a security concern, but its better if we can rely on the default region of the user's account. 
We can use the following URL instead. 

`https://console.aws.amazon.com/dynamodbv2`